Fix builds in the case of separate source from the build directory.

RP 2013/4/17

Upstream-Status: Pending
---
 pam-ck-connector/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
diff --git a/pam-ck-connector/Makefile.am b/pam-ck-connector/Makefile.am
index 7778d16..9b960a7 100644
--- a/pam-ck-connector/Makefile.am
+++ b/pam-ck-connector/Makefile.am
@@ -4,7 +4,7 @@ if ENABLE_PAM_MODULE
 
 AM_CFLAGS = 					\
 	$(LIBDBUS_CFLAGS)			\
-	-I$(top_builddir)/libck-connector	\
+	-I$(top_srcdir)/libck-connector	\
 	$(NULL)
 
 pamlibdir = $(PAM_MODULE_DIR)
